### PR TITLE
[stable/postgresql] Adds note about the postgis inclusion in the postgresql container image

### PR DESCRIPTION
**Pull request by rafariossaa**
#### What this PR does / why we need it:
Adds a note in the README about the inclusion of postgis in the postgresql container image.
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by rafariossaa** [comment]
Not merge it yet,  the images are being released.

**Comment by rafariossaa** [comment]
- Do not merge until fully reviewed and aceppted.

**Comment by rafariossaa** [comment]
We are working in double checking all is working fine.
This version using the postgresql container images for version 10, 11 and 12 that includes postgis 2.5
Postgis was compiled using following dependencies:
- protobuf
- protobuf-c
- json-c
- geos
- proj
- gedal

File: stable/postgresql/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 6.5.3
+version: 6.5.4
```
Comment (carrodher) [inline stable/postgresql/Chart.yaml]: Can you bump the minor version as it is a new feature?

File: stable/postgresql/README.md [hunk 2]
```diff
@@ -371,6 +371,16 @@ $ helm upgrade my-release bitnami/influxdb \
 
 > Note: you need to substitute the placeholders _[POSTGRESQL_PASSWORD]_, and _[REPLICATION_PASSWORD]_ with the values obtained from instructions in the installation notes.
 
+## 6.5.4
```
Comment (carrodher) [inline stable/postgresql/README.md]: This line should be updated according to the previous comment

**Comment by rafariossaa** [comment]
Got lot of unintended commits in this branch. 
Pushing a new one.

**Pull request closed without merging**